src/test/lib/EventTest.cpp

```diff
@@ -499,7 +499,7 @@ QuicTestValidateConnectionEvents3(
 
     QUIC_BUFFER* ResumptionTicket = nullptr;
     QuicTestPrimeResumption(
-        QuicAddrGetFamily(&ServerLocalAddr.SockAddr),
+        QuicAddrGetFamily(&ServerLocalAddr.SockAddr) == QUIC_ADDRESS_FAMILY_INET ? QUIC_ADDRESS_FAMILY_INET : QUIC_ADDRESS_FAMILY_INET6,
```

Comment:
Why?

Reply:
This is always 0, which does not work appropriately with duonic as it assert check the family whether 4 or 6.
Normal socket case just uses ipv6 localhost if it is 0. This change is for duonic to work with 0.